app/assets/javascripts/Components/submission_table.jsx

```diff
@@ -412,6 +437,28 @@ class RawSubmissionTable extends React.Component {
     );
   };
 
+  toggleShowInactiveGroups = (event, init) => {
```

Comment:
Okay this is better but not exactly what I meant. Don't pass in `event` or `init`, just pass in a boolean (either `true` or `false`).

During initialization you can pass in `false`; for the onclick/onchange, pass in the value of `event.target.checked`

Reply:
Ohh that makes a lot of sense! Will change.